src/axom/spin/internal/linear_bvh/BVH_impl.hpp

```diff
@@ -497,43 +488,49 @@ void BVH< NDIMS, ExecSpace, FloatType >::findPoints( IndexType* offsets,
   };
 
   // STEP 3: get counts
-  int total_count = bvh_get_counts< NDIMS,ExecSpace >(
-    leftPredicate, rightPredicate, inner_nodes, leaf_nodes,
-    numPts, counts, x, y, z );
+  int total_count = 0;
+  AXOM_PERF_MARK_SECTION( "PASS[1]:count_traversal", 
```

Comment:
> "PASS[1]:count_traversal"
"PASS[2]:fill_traversal"

Suggestion: Remove ``PASS[1]`` and ``PASS[2]`` from the annotations -- while this is useful for your current analysis, it might not be as useful to users / future use cases.

Reply:
I don't really expect users to dig in at this level of granularity.